OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -410,9 +410,30 @@ public interface IRenderAboveShroudWhenSelected
 		bool SpatiallyPartitionable { get; }
 	}
 
-	/// <summary>
-	/// Indicates target types as defined on <see cref="Traits.ITargetable"/> are present in a <see cref="Primitives.BitSet{T}"/>.
-	/// </summary>
+	public interface ISelection
+    {
+		int Hash { get; }
+		IEnumerable<Actor> Actors { get; }
+
+		void SetNotifySelection(IEnumerable<INotifySelection> worldNotifySelection);
+		void Add(Actor a);
+		void Remove(Actor a);
+		void OnOwnerChanged(Actor a, Player oldOwner, Player newOwner);
+		bool Contains(Actor a);
+		void Combine(World world, IEnumerable<Actor> newSelection, bool isCombine, bool isClick);
+		void Clear();
+		void Tick(World world);
+		void DoControlGroup(World world, WorldRenderer worldRenderer, int group, Modifiers mods, int multiTapCount);
+		void AddToControlGroup(Actor a, int group);
+		void RemoveFromControlGroup(Actor a);
+		int? GetControlGroupForActor(Actor a);
+		List<MiniYamlNode> Serialize();
+		void Deserialize(World world, List<MiniYamlNode> data);
+	}
+
+    /// <summary>
```

Comment:
This still has spaces, for example.

Reply:
And the two lines below (the whole summary block, basically).